vue3-apollo-quasar/src/stories/Introduction.stories.mdx

```diff
@@ -182,14 +182,22 @@ We recommend building UIs with a [**component-driven**](https://componentdriven.
       Configure, customize, and extend
     </span>
   </a>
-  <a className="link-item" href="https://storybook.js.org/tutorials/" target="_blank">
+  <a
+    className="link-item"
+    href="https://storybook.js.org/tutorials/"
```

Comment:
Check if the slash in the end needed. It will not show the link is visited.

Reply:
This came with storybook. Not us. Should be deleted soon